src/domManipulation.ts

```diff
@@ -221,22 +278,40 @@ export function togglePopup(customLat, customLong) {
     hideForecastLoader,
   );
 
-  loadMarker(customLat, customLong);
-  document.getElementById("main-cities-container").style.visibility = "hidden";
-  document.getElementById("main-cities-container").style.display = "none";
-  document.getElementById("popup-container").style.visibility = "visible";
-  document.getElementById("popup-container").style.display = "flex";
-  document.getElementById("custom-container").style.visibility = "visible";
-  document.getElementById("custom-container").style.display = "flex";
+  loadMarker(Number(customLat), Number(customLong));
+
+  const mainCitiesDisplay = document.getElementById("main-cities-container");
+
+  if (mainCitiesDisplay !== null) {
+    mainCitiesDisplay.style.visibility = "hidden";
+    mainCitiesDisplay.style.display = "none";
+  }
+  
+  const popupDisplay = document.getElementById("popup-container");
+
+  if (popupDisplay !== null) {
+    popupDisplay.style.visibility = "visible";
+    popupDisplay.style.display = "flex";
+  }
+  
+  const customForecastDisplay = document.getElementById("custom-container");
+
+  if (customForecastDisplay !== null) {
+    customForecastDisplay.style.visibility = "visible";
+    customForecastDisplay.style.display = "flex";
+  }
```

Comment:
make it one if statement (or guard clause)

Reply:
Updated implementation as a guard clause.